lib/simple/sql/config.rb

```diff
@@ -54,13 +54,18 @@ def database_url_from_database_yml(path)
 
   def load_activerecord_base_configuration(path:, env:)
     require "yaml"
-    opts = {}
-    opts[:aliases] = true if Psych::VERSION >= '4.0'
```

Comment:
aliases also worked prios to Psych 4, so I don't see need for this. (Definitely added in 3.1 which is now 5 years old)